### PR TITLE
Updated approach for installing nanomsg in copp TC

### PR DESCRIPTION
**Pull request by OleksandrKozodoi**
Signed-off-by: Oleksandr Kozodoi <oleksandrx.kozodoi@intel.com>

### Description of PR

Summary:
- Updated checking availability of the nanomsg in the syncd container 
- Added check Debian version before installation of nano
- Added approach for disabling bgp before installation of nano

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
syncd container was upgraded to Debian 11 Bullseye. test_copp uses Python 2 to install nano in syncd container, but Debian 11 doesn't support Python 2. #5905 PR of sonic-mgmt added approach for installing the correct version of Python, but this doesn't cover the case when python isn't available. So that approach was updated.
#### How did you do it?
Updated approach for installing nanomsg in copp TC
#### How did you verify/test it?
Run test cases. Tests passed.
```
copp/test_copp.py::TestCOPP::test_policer[cab18-4-dut-ARP] PASSED                                                                                                                   [  7%]
copp/test_copp.py::TestCOPP::test_policer[cab18-4-dut-IP2ME] PASSED                                                                                                                 [ 14%]
copp/test_copp.py::TestCOPP::test_policer[cab18-4-dut-SNMP] PASSED                                                                                                                  [ 21%]
copp/test_copp.py::TestCOPP::test_policer[cab18-4-dut-SSH] PASSED                                                                                                                   [ 28%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-BGP] PASSED                                                                                                                [ 35%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-DHCP] PASSED                                                                                                               [ 42%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-DHCP6] PASSED                                                                                                              [ 50%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-LACP] PASSED                                                                                                               [ 57%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-LLDP] PASSED                                                                                                               [ 64%]
copp/test_copp.py::TestCOPP::test_no_policer[cab18-4-dut-UDLD] PASSED                                                                                                               [ 71%]
copp/test_copp.py::TestCOPP::test_add_new_trap[cab18-4-dut] PASSED                                                                                                                  [ 78%]
copp/test_copp.py::TestCOPP::test_remove_trap[cab18-4-dut-delete_feature_entry] PASSED                                                                                              [ 85%]
copp/test_copp.py::TestCOPP::test_remove_trap[cab18-4-dut-disable_feature_status] PASSED                                                                                            [ 92%]
copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot[cab18-4-dut] PASSED                                                                                                 [100%]
```

### CODE REVIEW

**Comment by OleksandrKozodoi** [comment]
Hi, @ZhaohuiS. Could you please take a look?

**Comment by OleksandrKozodoi** [comment]
@ZhaohuiS May #5961 be closed?

**Comment by ZhaohuiS** [comment]
> Hi, @ZhaohuiS. Could you please take a look?

@OleksandrKozodoi I just reverted#5905 https://github.com/Azure/sonic-mgmt/pull/5961 since on some platform, it doesn't have python command.
For your change, what platform do you test?
And let me verify if it works on other platforms.

File: tests/copp/copp_utils.py [hunk 1]
```diff
@@ -205,25 +206,46 @@ def _install_nano(dut, creds,  syncd_docker_name):
...
+                    && mkdir -p build && cmake . && make install && ldconfig && cd .. && rm -rf nanomsg-1.0.0 \
+                    && rm -f 1.0.0.tar.gz && pip2 install cffi==1.7.0 && pip2 install --upgrade cffi==1.7.0 && pip2 install nnpy \
+                    && mkdir -p /opt && cd /opt && wget https://raw.githubusercontent.com/p4lang/ptf/master/ptf_nn/ptf_nn_agent.py \
+                    && mkdir ptf && cd ptf && wget https://raw.githubusercontent.com/p4lang/ptf/master/src/ptf/afpacket.py && touch __init__.py \
+                    " '''.format(http_proxy, https_proxy, syncd_docker_name)
+
+        try:
+            # Stop bgp sessions
+            dut.command("sudo config feature autorestart bgp disabled")
+            dut.command("sudo config feature state bgp disabled")
```
Comment (ZhaohuiS) [inline tests/copp/copp_utils.py:244]: @OleksandrKozodoi One question here, why need to stop bgp before installing nanomsg?

**Comment by ZhaohuiS** [comment]
@OleksandrKozodoi thank you for your fix. I verified it on different platforms, looks good. I only have a quick question, why stop bgp before installing nanomsg.

File: tests/copp/copp_utils.py (hunk 1 above)
Comment (OleksandrKozodoi) [reply tests/copp/copp_utils.py:244]: Those steps are used for installing necessary packages from the external area network

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Pull request merged**